R/Backend-class.R

```diff
@@ -166,7 +178,8 @@ setMethod(
     signature="Backend",
     definition=function(object, files, spectraData, ..., BPPARAM=bpparam()) {
     object@files <- normalizePath(files)
-    object
+    if (validObject(object))
```

Comment:
I just wonder if we really need the `if` here (we use it everywhere). `validObject` returns `TRUE` or throws an error. So we could save one if/comparison (I know that wouldn't save much time):
```r
validObject(object)
object
```
What do you think?

Reply:
makes sense.

Reply:
Did that now in the new commit.

Reply:
Hasn't change in this PR preview (in the line above).